functional_tests/steps/bundles.py

```diff
@@ -67,3 +69,63 @@ def the_selected_datasets_are_displayed(context: Context) -> None:
     context.page.get_by_text(
         "Deaths registered weekly in England and Wales by region (Edition: Example Dataset 3, Ver: 1)"
     ).is_visible()
+
+
+# bundle create amend
+@step('a bundle has been created with name "{bundle_name}" and creator "{creator}"')
+def a_bundle_has_been_created_with_user(context: Context, bundle_name: str, creator: str) -> None:
+    context.user = UserFactory.create(
+        username=creator, email="email@example.com", first_name="First_Name", last_name="Last_Name"
+    )
```

Comment:
We don't need to call `.create` or provide the name or email – those will be randomly generated for us anyway.

This can be simplified to:
```suggestion
    context.user = UserFactory(username=creator)
```

However, looking through your additions again I don't think you ever check of this exact username anyway, so do we need to provide it in the first place? I see that you grab the full name and username directly from the user object you create.

I think we can remove that parameter to keep things simple and just have:
```python
context.user = UserFactory()
```